.expeditor/verify.pipeline.yml

```diff
@@ -101,6 +103,8 @@ steps:
   commands:
     - /workdir/scripts/bk_tests/bk_container_prep.sh
     - zypper install -y cronie
+    - zypper install -y cron
```

Comment:
You shouldn't need cron if you have cronie, which is the default in opensuse. Also you can stick these all in a single line which runs faster.

Reply:
Thanks @tas50 , Going to update that.